docs/roman/introduction.rst

```diff
@@ -262,11 +245,11 @@ individual steps, you have two options:
     This option will save the step results using the name specified.
 
 For example, to save the result from the dark current step of
-``calwebb_detector1`` in a file named based on ``intermediate``, use
+``calroman_detector1`` in a file named based on ``intermediate``, use
```

Comment:
Does this add the ".asdf" or is the file just called "intermediate"? If the asdf is not added can we just call the file intermediate.asdf to be clear?